### ISSUE TITLE
False positive on W1648

### ISSUE THREAD

**Issue by sfermigier** [PyCQA/pylint#1608]
### Steps to reproduce

```
➜  abilian-sbe git:(master) ✗ cat toto.py
from __future__ import absolute_import

from .commands import titi

➜  abilian-sbe git:(master) ✗ pylint --py3k toto.py
************* Module toto
W:  3, 0: Module moved in Python 3 (bad-python3-import)

------------------------------------------------------------------
Your code has been rated at 5.00/10 (previous run: 5.00/10, +0.00)
```

### Current behavior

Gives a warning (believes we are importing from the `commands` module in the standard library.

### Expected behavior

Shouldn't give a warning (we're importing from our own code).

### pylint --version output

pylint 1.7.2,
astroid 1.5.3
Python 2.7.13 (default, Dec 18 2016, 07:03:34)
[GCC 4.2.1 Compatible Apple LLVM 7.0.2 (clang-700.1.81)]

**Comment by rogalski** [comment]
Thanks for submitting an issue.

In [`_ _warn_if_deprecated`](https://github.com/PyCQA/pylint/blob/master/pylint/checkers/python3.py#L629) we need to either skip emitting warning if import is _relative_ or maybe infer whether checked module is inferred as part of stdlib.
